File: ОП/Python/Лабораторна5_2/Лабораторна5_2/Лабораторна5_2.py
n=int(input("Enter quantity of prime numbers (n>=1)"))
num_now=2
kilk=1
print("Prime number is 2")
while kilk<n:
    num_now+=1
    dil=2
    while num_now/dil != num_now//dil and num_now != dil:
        dil+=1
    if num_now==dil:
        kilk+=1
        print("The prime number is", num_now)

File: ОП/C/Перший семестр/Лабораторна5_2/Лабораторна5_2/Лабораторна5_2.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n;
    int kilk = 1;
    int dil;
    
    cout << "enter quantity of prime numbers(n>=1)" << endl;
    cin >> n;
    cout << "Prime number is 2" << endl;
    for (int num_now = 3; kilk<n; num_now += 1) 
    {
        dil = 2;
        while (double(num_now) / dil != num_now / dil && num_now != dil) dil += 1;
        if (num_now == dil) 
        {
            kilk += 1;
            cout << "prime number is " << num_now << endl;
        }
    }   
 
    
    return 0;
}